### PR TITLE
Fix/article byline fix

### PR DESCRIPTION
**Pull request by nehasri89**
- Changing Line Height for byline as per design
- Updating snapshot
screen shot 2017-09-19 at 12 14 13
screen shot 2017-09-19 at 14 11 20

### CODE REVIEW

File: packages/article-byline/article-byline-styles.js [hunk 1]
```diff
@@ -1,10 +1,22 @@
 import { Platform } from "react-native";
 
+const lineHeightStyle = Platform.select({
+  ios: {
+    lineHeight: 14
+  },
+  android: {
+    lineHeight: 17
+  },
+  web: {
```
Comment (nehasri89) [inline packages/article-byline/article-byline-styles.js]: Needs
whiteSpace: "pre" style for web, not sure lineHeight: 13 resolves to 13px? need to check in storybook.

Comment (sericaia) [reply packages/article-byline/article-byline-styles.js]: `whiteSpace: "pre"` > where does it come from?

LineHeight resolves, I confirmed and you can check that on snapshots too

Comment (killanaca) [reply packages/article-byline/article-byline-styles.js]: it resolves to 13px because we use vanilla object attributes (not StyleSheet)

Comment (nehasri89) [reply packages/article-byline/article-byline-styles.js]: @sericaia the whitespace attr was originally there in the web styles file

Comment (sericaia) [reply packages/article-byline/article-byline-styles.js]: Fixed

**Pull request merged**